ui/components/MesheryResults.js

```diff
@@ -396,7 +396,8 @@ const mapStateToProps = (state) => {
   const startKey = state.get('results').get('startKey');
   const results = state.get('results').get('results').toArray();
   const results_selection = state.get('results_selection').toObject();
-  const user = state.get('user').toObject();
+  let user = state.get('user') ? state.get('user').toObject() : null;
```

Comment:
Is there any issue with this? `null` shouldn't be valid as Meshery isn't functional without authenticating whether it be remote provider or local provider, so what does this pr serve?
/cc @Nikhil-Ladha @leecalcote

Reply:
You are correct @kushthedude but, the error shouldn't be allowed to open any page without selecting the provider. But,, here on going to the error page the issue was coming up. 
So, Immanuel, what steps did you follow to come to this error?

Reply:
I noticed that the Results section when clicked on, shows an error instead of actually displaying the result if there is a user , or no result if there is no user.

So what I basically did was to add a conditional to correct that error.

Reply:
You chose a provider, right?

Reply:
@Nikhil-Ladha I wrote the steps I took to come to this error on the description.

Reply:
It doesn't states which provider you are selecting or you are selecting it all or not. That's why I asked here.

Reply:
Can you do one thing, try the steps @dhruv0000 mentioned and share a quick gif for the same?

Reply:
No I didn't chose a provider, I forked and cloned the Meshery project, going through it I discovered the Results tab when clicked on displays an errro, I debuded and found out the issue was in the line 399 of the Components.MesheryResults

Reply:
Okay, so what commands did you run before coming to this page?

Reply:
When I cloned the Meshery project. I cd into the UI and installed the node modules , then I opened the UI project . These are the following commands I ran on my terminal.
-cd ui 
-npm install 
-npm run dev ( which finally opened the Meshery UI dashboard on my browser ).

Then I started going through the performance, Results etc. Found out that the Results section displays an error, I debugged it and discobered the issue is on line 399 of the Components.MesheryResults.js

Reply:
Wwoohoo, sorry but these are not the steps you should follow for running Meshery UI😅.
You can follow the contributing guide , to run the UI. Then, I think as @kushthedude mentioned you won't be getting this error.

Reply:
Oh, I just checked it out. Actually I don't see the difference between the steps there in running the UI and what I wrote. Same localhost:3000 . 

Are you saying, from. Your end, you are not seeing the error?

Reply:
I only didn't signoff before committing and pushing. Which I would do today